### ISSUE TITLE
translations and update to newer version of workshopper 

### ISSUE THREAD

**Issue by gorhgorh** [tableflip/nodebot-workshop#20]
First nodeschool in paris is this weekend, I'll make a fast translation of the current ws content since i'm in a rush, but now the newer version of the workshopper have now l10n support i think we should update so we can decouple exercise and language and avoid package duplication. 

what do you think about it ?

**Comment by achingbrain** [comment]
Good idea.  Looks like there's a `LANG` environmental variable we can use. On my machine (OS X) it's value is `en_GB.UTF-8`.  Not sure about other platform support though - presumably Linux/Bash will be ok but not sure about Windows or other shell environments.

**Comment by castillobg** [comment]
I love the idea. I can translate the current material into neutral LatAm Spanish. @gorhgorh have you started the I10n porting or can I go ahead? :)

**Comment by olizilla** [comment]
https://github.com/workshopper/workshopper-adventure exists now and supports i18n outta tha box.
PORT ALL TEH THINGS!

**Comment by achingbrain** [comment]
Boom, done.  All filthy English strings pulled out into a [translation file](https://github.com/tableflip/nodebot-workshop/blob/master/i18n/en.json).

@gorhgorh @castillobg if you guys fancy doing French and Spanish translations that would be epic.

@n0bisuke do you think you could update the Japanese translation with the extra strings?

**Comment by gorhgorh** [comment]
hey hey will be glad to do that !

2016-04-22 17:15 GMT+02:00 Alex Potsides <notifications@github.com>:

**Comment by castillobg** [comment]
@achingbrain wooo! Will get started next week after exams. Thanks!

**Comment by n0bisuke** [comment]
@achingbrain I'm sorry, I was late. it's ok!

**Comment by forivall** [comment]
It looks like this issue can be closed now?

**Comment by olizilla** [comment]
Almost... @castillobgr Would you be up for writing a Spanish translation? No worries if you're too busy right now, PRs welcome anytime!

**Comment by castillobgr** [comment]
@olizilla I totally forgot about this :/ I'd be able to start in 2 weeks or so though :)
